client/src/legacy/TabSet.js

```diff
@@ -17,49 +21,15 @@ $.entwine('ss', function($){
 
       this._super();
     },
-
-    /**
-     * @func openTabFromURL
-     * @param {string} hash
-     * @desc Allows linking to a specific tab.
-     */
-    openTabFromURL: function (hash) {
-      var $trigger;
-
-      // Make sure the hash relates to a valid tab.
-      $.each(this.find('.ui-tabs-anchor'), function () {
-        // The hash in in the button's href and there is exactly one tab with that id.
-        if (this.href.indexOf(hash) !== -1 && $(hash).length === 1) {
-          $trigger = $(this);
-          return false; // break the loop
-        }
-      });
-
-      // If there's no tab, it means the hash is invalid, so do nothing.
-      if ($trigger === void 0) {
-        return;
-      }
-
-      // Switch to the correct tab when the page is loaded
-      $(() => {
-        $trigger.click();
-      });
-    },
-
-  }),
-
-    /**
-   * Lightweight wrapper around jQuery UI tabs for generic tab set-up
-   */
-  $('.ss-tabset').entwine({
-    IgnoreTabState: false,
-
+    
     onadd: function() {
       this.on("tabsactivate", (function (event, {newPanel}) {
         this.lazyLoadGridFields(newPanel);
```

Comment:
Remove this line and call it from inside `triggerLazyLoad()` instead. That way anyway calling `triggerLazyLoad()` from their own code (for whatever reason they may have) will not have to also call `lazyLoadGridFields()` separately.
```suggestion
```